vpr/src/route/multi_queue_priority_queue.cpp

```diff
@@ -19,23 +19,29 @@ void MultiQueuePriorityQueue::init_heap(const DeviceGrid& grid) {
     // TODO: Reserve storage for MQ_IO
 }
 
-bool MultiQueuePriorityQueue::try_pop(pq_node_t &pq_top) {
+bool MultiQueuePriorityQueue::try_pop(pq_prio_t &prio, RRNodeId &node) {
     auto tmp = pq_->tryPop();
     if (!tmp) {
         return false;
     } else {
-        pq_top = std::get<1>(tmp.get());
+        pq_index_t node_id;
+        std::tie(prio, node_id) = tmp.get();
+        node = RRNodeId(node_id);
```

Comment:
For this cast to make sense we need to ensure that the size of the RRNodeId is 32 bits.

Please add a static_assert enforcing that sizeof(RRNodeId) == sizeof(pq_index_t)

This will prevent bugs in the future.